boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+struct Imu;
+
+struct ImuData
```

Comment:
Struct shouldn't be declared like this in the .h file. This should just be `struct ImuData;`

Reply:
What if I want to reuse a struct in multiple files?

Reply:
The App Imu should be created inside the main.c file. If you want to use it in multiple files, you can get it from the world.

Reply:
> Struct shouldn't be declared like this in the .h file. This should just be `struct ImuData;`

Actually struct ImuData should be removed entirely

Reply:
Yeah ImuData is only used in IO code now, I removed it yesterday